interface/forms/gad7/view.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+/**
+ * gad-7 form using form api     view.php
+ * open a previously completed GAD-7 form for further editing
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+include_once("gad7.inc");  // common strings etc
+use OpenEMR\Common\Csrf\CsrfUtils;    // security module
+use Mpdf\Mpdf;  /* used to generate a pdf of the form */
+/*  strings used in menus */
+$menu_strings_array = array (0 => 'Not at all',1 => 'Several days',2 => 'More than half the days',3 => 'Nearly every day');
+$menu_difficulty_array = array (0 => 'Not at all',1 => 'Somewhat difficult',2 => 'Very difficult',3 => 'Extremely difficult');
+formHeader("Form: Gad-7");
+$returnurl = 'encounter_top.php';   // ?? update top of encounter window when done - maybe
+?>
+<html><head>
+ <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+</head>
```

Comment:
same as above comment

Reply:
thanks, have used core header and pushed the mods, checks being done...